apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -97,6 +106,27 @@ export const folderRouter = router({
               throw err
             })
 
+          const indexPageBlob = await tx
+            .insertInto("Blob")
+            .values({
+              content: jsonb(createIndexPage(folderTitle)),
+            })
+            .returning("id")
+            .executeTakeFirstOrThrow()
+
+          await tx
+            .insertInto("Resource")
+            .values({
+              parentId: folder.id,
+              draftBlobId: indexPageBlob.id,
+              title: folderTitle,
+              type: ResourceType.IndexPage,
+              permalink: INDEX_PAGE_PERMALINK,
+              siteId,
+            })
+            .returning(["id", "draftBlobId"])
+            .executeTakeFirstOrThrow()
+
```

Comment:
issue: might want to also do the audit log for the creation of the index page

Reply:
yeap, done